packages/core/upload/admin/src/components/AssetCard/ImageAssetCard.js

```diff
@@ -3,21 +3,20 @@ import React from 'react';
 import { CardAsset } from '@strapi/design-system';
 import PropTypes from 'prop-types';
 
-import { appendSearchParamsToUrl } from '../../utils';
+import { cacheBustUrl } from '../../utils';
 
 import { AssetCardBase } from './AssetCardBase';
 
-export const ImageAssetCard = ({ height, width, thumbnail, size, alt, ...props }) => {
-  // Prevents the browser from caching the URL for all sizes and allow react-query to make a smooth update
-  // instead of a full refresh
-  const urlWithCacheBusting = appendSearchParamsToUrl({
+export const ImageAssetCard = ({ height, width, thumbnail, size, alt, isUrlSigned, ...props }) => {
+
+  const thumbnailUrl = isUrlSigned ? thumbnail : cacheBustUrl({
     url: thumbnail,
-    params: { updatedAt: props.updatedAt },
+    timestamp: props.updatedAt,
   });
```

Comment:
@Fryuni this change specifically, is completely pointless now & just adds noise to understanding the code

Reply:
@joshuaellis The url is only cache busted if it is signed therefore the variable cannot be named urlWithCacheBusting anymore as it won't be accurate.  If I leave it like this:
`const thumbnailUrl = isUrlSigned ? thumbnail : appendSearchParamsToUrl({ ... })`
reading the code doesn't explain the reason for appending the param, unless it's written in a comment above. I opted for creating a method with a self-explanatory name and docs&tests. In my opinion this brings more clarity.

Reply:
i was referring to the other point of adding in a function called `cacheBustUrl` that is a wrapper around an existing function.

Reply:
I don't agree it brings more clarity, it adds _another_ function that needs _another_ test only to wrap an existing function. imo the adding `isUrlSigned` and checking that _before_ appending a search param is probably enough. I need to check with BE before this PR can be approved anyway, but let's correct this first.

Reply:
@joshuaellis ok, we can agree to disagree..  if I'm honest I find your responses unkind, unclear and a bit patronising. We're here to learn from each other and build something together, let's not forget that.

I removed the function and amended the inline comment.